test/helpers/supertest_session/session-data/base.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('lodash');
+
+module.exports = (appName, overridenPages) => {
+  const possibleSteps = require(`./${appName}/steps`);
+
+  return (stepOrData, data) => {
+    const props = {};
+    if (typeof stepOrData === 'string') {
+      props.steps = _.dropRightWhile(possibleSteps, val => val !== stepOrData);
+      props.steps.pop();
+    } else {
+      props.steps = possibleSteps;
+    }
+    props.steps.forEach(prop => {
+      if (prop !== '/') {
+        try {
+          /* eslint-disable no-nested-ternary */
+          if (overridenPages && require(`./${appName}/${overridenPages}${prop}`)) {
+            Object.assign(props, require(`./${appName}/${overridenPages}${prop}`),
+              data ? data : typeof stepOrData === 'object' ? stepOrData : {});
+          } else {
+            Object.assign(props, require(`./${appName}/pages${prop}`),
+              data ? data : typeof stepOrData === 'object' ? stepOrData : {});
+          }
+
+          /* eslint-disable no-empty */
```

Comment:
I would do then you don't need it twice
```suggestion
          /* eslint-disable no-empty */
```

Reply:
I have amended this now